docs/framework/contributing/contributing.md

```diff
@@ -95,7 +95,7 @@ Some additional things you should keep in mind:
 
 CKEditor&nbsp;5 is a project with global impact, so contributing translations is both an easy and powerful way to help.
 
-We use the Transifex service for translations at the following address: [https://www.transifex.com/ckeditor/ckeditor5/dashboard/](https://www.transifex.com/ckeditor/ckeditor5/dashboard/).
+To make CKEditor&nbsp;5 available in multiple languages, we use `.po` files for storing translations. These are text files commonly used in software development to translate interface texts. Each CKEditor&nbsp;5 package keeps translation files in the `lang/translations` directory. To provide missing translations, just edit the `.po` file and prepare a [pull request](#creating-a-pull-request).
```

Comment:
SHould these be edited by hand, actually?

Reply:
Yeah, starting the upcoming release, `*.po` files can be updated directly by users via PR.

Reply:
Cool.